versioneer.py

```diff
@@ -0,0 +1,2140 @@
+
```

Comment:
not https://github.com/python-versioneer/python-versioneer ? i know nothing about this just seemed like 'gotta be a gh action for this'.  did you modify it?

Reply:
RTFM -- this is how python-versioneer works